pages/Login/index.tsx

```diff
@@ -22,106 +25,138 @@ const Login: NextPage = () => {
   } = useForm<ILogin>();
   const router = useRouter();
   const { data, mutate, ...dataQuery } = useLogin();
+  const [isError, setError] = useState(false);
+  const [value, setValue] = useState(0);
 
   const onSubmit: SubmitHandler<ILogin> = (dataForm) => {
     const success = () => {
       router.push(Routes.Home);
     };
-    mutate(dataForm, { onSuccess: success });
+    const error = () => {
```

Comment:
no need for this, react query already provides a boolean for this in the useMutation return object

Reply:
Okkk